guides.xml

```diff
@@ -3,6 +3,16 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="https://www.phpdoc.org/guides packages/guides-cli/resources/schema/guides.xsd">
     <project title="phpDocumentor Guides"/>
+    <html_theme>bootstrap</html_theme>
+    <input>docs</input>
+    <output>output</output>
+    <input_format>rst</input_format>
+    <output_formats>
+        <output_format>html</output_format>
+    </output_formats>
+    <log_path>php://stder</log_path>
+    <fail_on_log>false</fail_on_log>
+    <show_progress>true</show_progress>
```

Comment:
In xml, it's allowed to replace underscores with dashes, which is much more common in XML attribute and element names (e.g. `fail-on-log`).

Also, it's common to use attributes instead of elements when dealing with scalar nodes (instead of array ones): `<guides fail-on-log="false">`

Reply:
However the existing tags base_template_paths and html_theme also use underscores?

Reply:
I tried renaming them but as soon as I rename them they do not work anymore and I get:

PHP Fatal error:  Uncaught Symfony\Component\Config\Definition\Exception\InvalidConfigurationException: Unrecognized option "input_format" under "guides". Did you mean "input-for
mat"? in /var/www/html/vendor/symfony/config/Definition/ArrayNode.php:327
Stack trace:
#0 /var/www/html/vendor/symfony/config/Definition/BaseNode.php(385): Symfony\Component\Config\Definition\ArrayNode->normalizeValue(Array)
#1 /var/www/html/vendor/symfony/config/Definition/Processor.php(32): Symfony\Component\Config\Definition\BaseNode->normalize(Array)

Reply:
Sorry, it's the other way around. You use `->scalarNode('input_format')` in the configuration builder, but `input-format` in the XML code.
The xml loader will automatically replace the dashes with underscores before processing it.